resources/js/features/user/settings/components/ChangePasswordSectionCard/ChangePasswordSectionCard.test.tsx

```diff
@@ -0,0 +1,60 @@
+import userEvent from '@testing-library/user-event';
+import axios from 'axios';
+
+import { render, screen } from '@/test';
+
+import { ChangePasswordSectionCard } from './ChangePasswordSectionCard';
+
+describe('Component: ChangePasswordSectionCard', () => {
+  it('renders without crashing', () => {
+    // ARRANGE
+    const { container } = render(<ChangePasswordSectionCard />, {
+      pageProps: {},
+    });
+
+    // ASSERT
+    expect(container).toBeTruthy();
+  });
+
+  it('given the user attempts to submit without a matching password and confirm password, does not submit', async () => {
+    // ARRANGE
+    vi.spyOn(window, 'confirm').mockImplementationOnce(() => true);
+    const putSpy = vi.spyOn(axios, 'put');
+
+    render(<ChangePasswordSectionCard />, {
+      pageProps: {},
+    });
+
+    // ACT
+    await userEvent.type(screen.getByLabelText(/current password/i), '12345678');
+    await userEvent.type(screen.getByLabelText(/new password/i), '87654321');
+    await userEvent.type(screen.getByLabelText(/confirm password/i), 'aaaaaaaa');
+
+    // ASSERT
+    expect(putSpy).not.toHaveBeenCalled();
+  });
+
+  it('given the user attempts to make a valid form submission, submits the request to the server', async () => {
+    // ARRANGE
+    vi.spyOn(window, 'confirm').mockImplementationOnce(() => true);
+    const putSpy = vi.spyOn(axios, 'put').mockResolvedValueOnce({ success: true });
+
+    render(<ChangePasswordSectionCard />, {
+      pageProps: {},
+    });
+
+    // ACT
+    await userEvent.type(screen.getByLabelText(/current password/i), '12345678');
+    await userEvent.type(screen.getByLabelText(/new password/i), '87654321');
+    await userEvent.type(screen.getByLabelText(/confirm password/i), '87654321');
+
+    await userEvent.click(screen.getByRole('button', { name: /update/i }));
+
+    // ASSERT
+    expect(putSpy).toHaveBeenCalledWith(route('settings.password.update'), {
+      currentPassword: '12345678',
+      newPassword: '87654321',
+      confirmPassword: '87654321',
+    });
+  });
```

Comment:
I have concerns that a failed update attempt will not be noticed by the user. The new message is not plastered across the top of the screen like the old one was, and fades fairly quickly.
![image](https://github.com/user-attachments/assets/7878b5fd-72d6-425e-a31e-a9bdd7ad40ea)

Reply:
I understand your concerns.

In latest, I've made two changes to try to help with this. First, I've increased the default duration of all toasts by 1 more second. I've also improved the contrast of all toasts so it's more obvious when one appears:

**Before**
![Screenshot 2024-09-02 at 3 55 31 PM](https://github.com/user-attachments/assets/51e52eec-cc39-4b2b-9b89-43ccabbd0a96)


**After**
![Screenshot 2024-09-02 at 3 55 22 PM](https://github.com/user-attachments/assets/c72ae2ca-d1fb-42c5-8f5f-3ce8b3ba84f0)